docker-compose.yml

```diff
@@ -13,7 +20,10 @@ services:
       PYLINTHOME: /app/.pylint.d
     env_file:
       - env.d/development/common
+      - env.d/development/postgresql
     ports:
       - "8090:8000"
     volumes:
       - .:/app
+    depends_on:
+      - "${DB_HOST:-postgresql}"
```

Comment:
Can you explain why using this environment variable ?

Reply:
Oh, I took it from a project having multiple database containers (mysql + postgresql) and it allows to switch between the two.
Since there is only postgresql here, i'll replace this with `- "postgresql"`.